Sources/Sentry/SentryTimeToDisplayTracker.m

```diff
@@ -143,12 +143,23 @@ - (void)finishSpansIfNotFinished
     }
 
     if (self.fullDisplaySpan.isFinished == NO) {
+        if (_fullyDisplayedReported) {
+            SENTRY_LOG_DEBUG(
+                @"SentrySDK.reportFullyDisplayed() was called but didn't receive a new frame to "
+                @"finish the TTFD span. Finishing the full display span so the SDK can start a new "
+                @"time to display tracker.");
+            [self.fullDisplaySpan finish];
```

Comment:
`m`: Not sure if I have enough insights into TTID/TTFD and frame tracking, but shouldn't we also remove the listener here as we do in L162?

Reply:
Good catch yes. I will add this at the top of the method.